bookserver/session.py

```diff
@@ -45,6 +46,10 @@ async def load_user(user_id: str):
     return await fetch_user(user_id)
 
 
+# The ``user_loader`` decorator doesn't propagate type hints. Fix this manually.
+load_user = cast(Callable[[str], Awaitable[AuthUserValidator]], _load_user)
```

Comment:
Ugh!!  This is where mypy goes completely off the rails IMHO.  Something that should not get in your way but add value by ensuring type correctness has now made the code very obscure.

Reply:
I agree, though part of the blame goes to FastAPI-LoginManager, since that's what fails to propagate type hints.